azure/terraform/vnetpeerings.rego

```diff
@@ -3,7 +3,7 @@ package rule
 # https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/virtual_network_peering
 
 #
-# PR-AZR-0075-TRF
+# PR-AZR-TRF-NTW-004
```

Comment:
@shahinshirvani it should be `VNP `or `AVNP`

Reply:
we have NTW for network watcher let's use NET for anything network related

Reply:
fixed